src/Sylius/Bundle/WebBundle/Resources/views/Backend/Order/show.html.twig

```diff
@@ -130,10 +130,10 @@
         <tr class="taxes">
             <td colspan="2">
                 <h5>{{ 'sylius.order.taxes'|trans }}</h5>
-                {% if not order.adjustments(taxAdjustment).isEmpty() %}
+                {% if order.allAdjustments(taxAdjustment) %}
                     <ul class="list-unstyled">
-                        {% for taxAdjustment in order.adjustments(taxAdjustment) %}
-                            <li>{{ taxAdjustment.description }} {{ taxAdjustment.amount|sylius_price(order.currency) }} </li>
+                        {% for tax in order.allAdjustments(taxAdjustment) %}
```

Comment:
micro-optimisation but might be worth setting order.allAdjustments(taxAdjustment) to a variable since you use it twice (repeated pattern below)